py3status/modules/apt_updates.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+Display number of pending updates for Ubuntu based Distros.
+
+Thanks to Iain Tatch <iain.tatch@gmail.com> for the script that this is based on.
+
+This will display a count of how many 'apt' updates are waiting to be installed.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds (default 600)
+    format: Display format to use
+        (default 'UPD: {apt}')
+    no_updates: Display format to use if there are no updates
+
+Format placeholders:
+    {apt} Number of pending apt updates
+
+Requires:
+    apt: Needed to display pending 'apt' updates
+
+@author Joshua Pratt <jp10010101010000@gmail.com>
+@license BSD
+
+SAMPLE OUTPUT
+{'full_text': 'UPD: 5'}
+"""
+
+import subprocess
+import sys
+
+FORMAT = 'UPD: {apt}'
+NO_UPDATES = 'UPD'
```

Comment:
We don't tend to do this in other modules (python string formatting knowledge is not assumed). Should these other modules also be updated? I'll make the change in the mean time.